aws-athena/views/default-vw_pin_sale.sql

```diff
@@ -286,45 +288,40 @@ SELECT
     unique_sales.num_parcels_sale,
     unique_sales.buyer_name,
     unique_sales.sale_type,
-    sale_filter.sale_filter_lower_limit,
-    sale_filter.sale_filter_upper_limit,
-    sale_filter.sale_filter_count,
+    unique_sales.sale_filter_same_sale_within_365,
+    unique_sales.sale_filter_less_than_10k,
+    unique_sales.sale_filter_deed_type,
+    COALESCE(sales_val.sv_is_outlier, FALSE) AS sale_filter_is_outlier,
```

Comment:
**suggestion:** Add an inline comment clarifying the difference between this and `sv_is_outlier` + a justification for it. I'll add the same to the docs.